pkg/argo-actions/upload_type_instances.go

```diff
@@ -119,7 +119,32 @@ func (u *Upload) render(payload *graphqllocal.CreateTypeInstancesInput, values m
 			return ErrMissingTypeInstanceValue(*typeInstance.Alias)
 		}
 
-		typeInstance.Value = value
+		// render value
+		if _, ok = value["value"]; ok {
+			typeInstanceValue, ok := value["value"].(map[string]interface{})
+			if !ok {
+				return ErrTypeConversion("value", "map[string]interface{}", *typeInstance.Alias)
+			}
+			typeInstance.Value = typeInstanceValue
+		} else {
+			// for backward compatibility, if there is an artifact without value/backend syntax,
+			// treat it as a value for TypeInstance
```

Comment:
it is mainly for the case like [here](https://github.com/capactio/hub-manifests/blob/eb48ac7e06d525051de6bce4f99f95ade5b44b2e/manifests/implementation/postgresql/create-user.yaml#L78-L81)